inc/item_devices.class.php

```diff
@@ -475,17 +475,17 @@ static function getDeviceForeignKey() {
     * @param $item
     * @param $table
     * @param $options            array
-    * @param $delete_all_column          (default NULL)
+    * @param $delete_all_column
     * @param $common_column
     * @param $specific_column
-    * @param $delete_column               (default NULL)
+    * @param $delete_column
     * @param $dynamic_column
    **/
    function getTableGroup(CommonDBTM $item, HTMLTableMain $table, array $options,
-                          HTMLTableSuperHeader $delete_all_column=NULL,
+                          HTMLTableSuperHeader $delete_all_column,
```

Comment:
I'm not sure about the deletion of theses null default values

It seems the null values are tested in: 
- for $delete_all_column: https://github.com/glpi-project/glpi/blob/master/inc/htmltablecell.class.php#L82
- for $delete_column: https://github.com/glpi-project/glpi/blob/master/inc/htmltablebase.class.php#L133